src/main/java/hu/oe/nik/szfmv/automatedcar/Main.java

```diff
@@ -16,24 +16,20 @@
     private World world;
 
     public static void main(String[] args) {
-
         new Main().run();
-
     }
 
     public void run() {
         init();
         loop();
-
     }
 
     private void init() {
         // create the world
         world = new World(5000, 3000);
         // create an automated car
-        car = new AutomatedCar(20, 20, "car_2_white.png");
+        car = new AutomatedCar(100, 300,"kek.png");
```

Comment:
[](https://pics.me.me/english-motherfucker-do-you-speak-it-13596485.png)

Reply:
Got it!